libbitstream/bitstream.c

```diff
@@ -177,17 +177,8 @@ STATIC void *opae_bitstream_parse_metadata(const char *metadata,
 	// version to 640/650 respectively.
 	// Allow 640/650 to serve as an alias for 1.
 	case 650:
```

Comment:
Don't need a fallthrough here?

Reply:
It didn't seem to require it.  This was with gcc 8.x on Fedora 29.